deploy/kubernetes/images/stable/image.yaml

```diff
@@ -51,5 +51,5 @@ imageTag:
   name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
   # Don't change stable image without changing pdImagePlaceholder in
   # test/k8s-integration/main.go
-  newTag: "v1.0.0-gke.0"
+  newTag: "v1.1.0-gke.0"
```

Comment:
All the sidecars should also be upgraded, and this patch should go into the base overlay: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/overlays/prow-gke-release-staging-rc/sidecar-upgrade.yaml